addons/web_editor/static/src/js/editor/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -1,6 +1,6 @@
 /** @odoo-module **/
 import { patienceDiff } from './patienceDiff.js';
-import { getRangePosition } from '../utils/utils.js';
+import { closestElement, getRangePosition } from '../utils/utils.js';
```

Comment:
Regarding the commit message: "caused problem in collaborative" - can you say more specifically what the problem is? This states what was changed on a technical level but not _why_ that change was needed.

Reply:
I have updated the commit message.